src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheck.java

```diff
@@ -641,6 +704,9 @@ private static Details getDetails(DetailAST ast) {
                 case TokenTypes.LITERAL_SWITCH:
                     details = getDetailsForSwitch(ast);
                     break;
+                case TokenTypes.VARIABLE_DEF:
+                    details = getDetailsForAnonymousInnerClass(ast);
```

Comment:
@nrmancuso 
@romani 

This needs correction (line 708). Since I am using VARIABLE_DEF, the method name shouldn't be ``getDetailsForAnonymousInnerClass(ast)`` but please let me know.

Reply:
Why do we need VARIABLE_DEF? Is OBJBLOCK not enough?

Reply:
@nrmancuso DONE.